lib/aptible/cli/subcommands/deploy.rb

```diff
@@ -29,6 +29,12 @@ def self.included(thor)
                    desc: 'Detach this app from its git repository: ' \
                          'its Procfile, Dockerfile, and .aptible.yml will be ' \
                          'ignored until you deploy again with git'
+            option :container_count, type: :numeric
+            option :container_size, type: :numeric
+            option :container_profile, type: :string,
+                                       desc: 'This option only effects new ' \
+                                             'services, not existing one.  ' \
+                                             'Examples: m5 c5 r5'
```

Comment:
The container count and size options need descriptions too. Also, typo:

```suggestion
            option :container_count, type: :numeric
            option :container_size, type: :numeric
            option :container_profile, type: :string,
                                       desc: 'This option only effects new ' \
                                             'services, not existing ones. ' \
                                             'Examples: m5 c5 r5'
```